server/base/src/main/java/org/apache/accumulo/server/conf/store/impl/ZooPropLoader.java

```diff
@@ -63,6 +63,9 @@ public ZooPropLoader(final ZooReaderWriter zrw, final VersionedPropCodec propCod
 
       Stat stat = new Stat();
       byte[] bytes = zrw.getData(propStoreKey.getPath(), propStoreWatcher, stat);
+      if (bytes.length == 0) {
```

Comment:
Addressed in ac8d0b8da8 - in this case, the ZooKeeper stat is also being used, so that should be another reliable check for the data length.  The NoNodeException likely was sufficient - but checking the stat does not seem to hurt.